modules/claims_api/app/sidekiq/claims_api/service_base.rb

```diff
@@ -36,6 +36,29 @@ class ServiceBase
 
     protected
 
+    def deep_dup_and_freeze_form_data(form_data_object, visited = {})
```

Comment:
Just calling `.dup.freeze` on the object did not preserve it.  Unfortunately, I had to go all the way to this point to get it it actually "preserve".  Feels like way too much needed to achieve that, but I was unable to find anything else that worked.  If anyone has any better options I am all ears on this.